### PR TITLE
 Add project documentation for Rata_Mircea_Andrei

### PR DESCRIPTION
**Pull request by MirceaAndrei**
### Pull Request Overview

This pull request adds/changes/fixes the documentation of the project.

### Testing Strategy

This pull request was tested by Rata Mircea Andrei.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Rata Mircea Andrei <mircea.radu2003@gmail.com>

### CODE REVIEW

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template.

File: docs/project/2024/Mircea_Andrei_RAŢĂ/index.md
Comment (alexandruradovici) [inline docs/project/2024/Mircea_Andrei_RAŢĂ/index.md:1]: Rename the folder to your username.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Mircea_Andrei_RAŢĂ/index.md
Comment (MirceaAndrei) [reply docs/project/2024/Mircea_Andrei_RAŢĂ/index.md:1]: The folder has my username and I also update the pull request template.

File: docs/project/2024/Mircea_Andrei_RAŢĂ/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+This combination of hardware and functionality results in a sophisticated yet user-friendly alarm system that offers reliable security for a variety of settings. It provides users with real-time alerts and control over the system, making it a valuable asset for safeguarding properties and ensuring peace of mind.
+
+
+## Motivation
+
+The motivation behind this project stems from a desire to enhance home and property security by leveraging modern technology. Traditional alarm systems can be costly and complex to install, often requiring professional services. In contrast, this project offers an opportunity to create an affordable and customizable smart alarm system using a Raspberry Pi Pico W and a range of hardware components.
+
+## Architecture 
+
+![System Diagram](https://i.imgur.com/VssvqIb.png)
```
Comment (alexandruradovici) [inline docs/project/2024/Mircea_Andrei_RAŢĂ/index.md:33]: Please add the diagram to your folder.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Mircea_Andrei_RAŢĂ/index.md (hunk 1 above)
Comment (MirceaAndrei) [reply docs/project/2024/Mircea_Andrei_RAŢĂ/index.md]: I added the diagram on my folder.

File: docs/project/2024/Mircea_Andrei_RAŢĂ/arhitecture.png
Comment (alexandruradovici) [inline docs/project/2024/Mircea_Andrei_RAŢĂ/arhitecture.png:1]: Please rename the folder to your curs.upb.ro username.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Mircea_Andrei_RAŢĂ/index.md
Comment (alexandruradovici) [reply docs/project/2024/Mircea_Andrei_RAŢĂ/index.md:1]: Your username is the part in front if @stud....

File: docs/project/2024/Mircea_Andrei_RAŢĂ/arhitecture.png
Comment (alexandruradovici) [inline docs/project/2024/Mircea_Andrei_RAŢĂ/arhitecture.png:1]: Delete this file.

File: docs/project/2024/Mircea_Andrei_RAŢĂ/index.md
Comment (alexandruradovici) [inline docs/project/2024/Mircea_Andrei_RAŢĂ/index.md:1]: Delete this file and folder.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Mircea_Andrei_RAŢĂ/index.md
Comment (MirceaAndrei) [reply docs/project/2024/Mircea_Andrei_RAŢĂ/index.md:1]: I have an issue. On my repository i have only the mircea_andrei.rata folder. I dont have the one that i need to delete ,the Mircea_Andrei_RAŢĂ.What should i do?

Comment (MirceaAndrei) [reply docs/project/2024/Mircea_Andrei_RAŢĂ/index.md:1]: I deleted the file and folder.

File: docs/project/2024/mircea_andrei.rata/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# Burglar alarm with notification via email
+
+ A smart burglar alarm system , equipped with motion detection, email alerts and keypad for security.
```
Comment (alexandruradovici) [inline docs/project/2024/mircea_andrei.rata/index.md:3]:
```suggestion
 A smart burglar alarm system, equipped with motion detection, email alerts and keypad for security.
```

**Review by alexandruradovici — DISMISSED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**